android/sdk/src/main/java/org/jitsi/meet/sdk/dropbox/Dropbox.java

```diff
@@ -0,0 +1,194 @@
+    }
+
+
+    /**
+     * Resolves the current user dropbox display name.
+     *
+     * @param token A dropbox access token.
+     * @param promise The promise used to return the result of the auth flow.
+     */
+    @ReactMethod
+    public void getDisplayName(final String token, final Promise promise) {
+        DbxRequestConfig config
+            = DbxRequestConfig.newBuilder(clientId).build();
+        DbxClientV2 client = new DbxClientV2(config, token);
+        // Get current account info
+        try {
+            FullAccount account = client.users().getCurrentAccount();
+            promise.resolve(account.getName().getDisplayName());
+        } catch (DbxException e) {
+            promise.reject(e);
+        }
+
+    }
+
+    /**
+     * Resolves the current user space usage.
+     *
+     * @param token A dropbox access token.
+     * @param promise The promise used to return the result of the auth flow.
+     */
+    @ReactMethod
+    public void getSpaceUsage(final String token, final Promise promise) {
+        DbxRequestConfig config
+            = DbxRequestConfig.newBuilder(clientId).build();
+        DbxClientV2 client = new DbxClientV2(config, token);
+        try {
+            SpaceUsage spaceUsage = client.users().getSpaceUsage();
+            WritableMap map = Arguments.createMap();
+            map.putString("used", String.valueOf(spaceUsage.getUsed()));
+            SpaceAllocation allocation = spaceUsage.getAllocation();
+            long allocated = 0;
+            if(allocation.isIndividual()) {
+                allocated += allocation.getIndividualValue().getAllocated();
+            }
+
+            if(allocation.isTeam()) {
+                allocated += allocation.getTeamValue().getAllocated();
+            }
+            map.putString("allocated", String.valueOf(allocated));
+            promise.resolve(map);
+        } catch (DbxException e) {
+            promise.reject(e);
+        }
+    }
+
+
+
+    /**
+     * Generate a client identifier for the dropbox sdk.
+     *
+     * @returns a client identifier for the dropbox sdk.
+     */
+    private String generateClientId() {
+        String clientId = "";
+        Context context = getReactApplicationContext();
+        PackageManager packageManager = context.getPackageManager();
+        ApplicationInfo applicationInfo = null;
+        PackageInfo packageInfo = null;
+
+        try {
+            String packageName = context.getPackageName();
+
+            applicationInfo
+                    = packageManager.getApplicationInfo(packageName, 0);
+            packageInfo = packageManager.getPackageInfo(packageName, 0);
+        } catch (PackageManager.NameNotFoundException e) {
+        }
+
+        if (applicationInfo == null) {
+            clientId += "JitsiMeet";
+        } else {
+            clientId
+                    += packageManager.getApplicationLabel(applicationInfo)
+                    .toString().replaceAll("\\s", "");
+        }
```

Comment:
@hristoterezov, what's the idea of declaring `String clientId = ""` way up there at the beginning at using it only later on. Please keep them together i.e. don't introduce a variable to the dev before they have to keep it in they mind context.

Also, that's a ternary right there. Much shorter, no second thoughts whether there's anything else in the `if`. And doesn't need a concatenation when combined with the var declaration into a var definition.